scripts/build/Platform/Android/build_config.json

```diff
@@ -29,7 +29,7 @@
   "debug": {
     "TAGS":[
         "nightly-incremental",
-        "nightly-clean",
+        "weekly-clean-ly",
```

Comment:
I think we can omit the `-ly` in the weekly tag since there will by only one weekly pipe. 

For the separate nightly pipe, I recommend we use a different identifier instead of `ly`. Something more generic like `-internal` or something.

Reply:
Agree. We want to be clear about that, unless the `-ly` is actually a filter on what environment it's running in